### PR TITLE
rpcinfo: add page

### PR DESCRIPTION
**Pull request by pxgamer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/windows/rpcinfo.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- List all programs registered on the local computer:
+
+`rpcinfo`
+
+- List all programs registered on a remote computer:
+
+`rpcinfo /p {{computer_name}}`
+
+- Call a specific program on a remote computer using TCP:
```
Comment (agnivade) [inline pages/windows/rpcinfo.md]: What's the default protocol ? We should mention that in parentheses.

Comment (mebeim) [reply pages/windows/rpcinfo.md]: No default as far as I can tell from the Microsoft doc.

Comment (agnivade) [reply pages/windows/rpcinfo.md]: Well, there has to be _some_ default protocol which is used. Strange that it's not mentioned in the docs.

Comment (pxgamer) [reply pages/windows/rpcinfo.md]: I can't find any information on this at all, so just installing it to test. 👍

Comment (mebeim) [reply pages/windows/rpcinfo.md]: @agnivade the point is that from the doc it seems that you cannot connect to a remote computer if you don't specify the protocol. If you look at the doc, you can see `[/t <Node Program> [<version>]]`, there is no `<Node Program>` outside square brackets that are already including `/t` or `/u`.

Comment (agnivade) [reply pages/windows/rpcinfo.md]: Oh I see ! Okay with me then

**Pull request merged**